src/Cake.Svn/Info/SvnInfoSettings.cs

```diff
@@ -10,16 +10,16 @@ public sealed class SvnInfoSettings : SvnRemoteSettings
         /// </summary>
         public SvnInfoSettings()
         {
-            Depth = SvnDepth.Infinity;
             RetrieveActualOnly = true;
             RetrieveExcluded = true;
-            Revision = -1;
         }
 
         /// <summary>
         /// Gets or sets the value of the revision on which Subversion should operate.
+        /// Use <c>null</c> for the default revision, <c>-1</c> for the head revision and numbers 
+        /// bigger than 0 for a cartain revision.
```

Comment:
What about 0?

Reply:
Should bigger than -1 I will correct this